src/mctp-bench.c

```diff
@@ -46,13 +73,43 @@ struct recv_ctx {
 	int sd;
 };
 
+struct command_response {
+	// A command_response value
+	uint8_t status;
+} __attribute__((packed));
+
+struct command_request_bench {
+	uint32_t flags;
+	uint16_t payload_size;
+	uint64_t message_count;
+} __attribute__((packed));
+
 static const size_t MSG_HEADER_LEN = sizeof(struct msg_header);
 static const size_t MAX_LEN = 64 * 1024 - 1;
 static const uint32_t SEQ_START = UINT32_MAX - 5;
 static const uint16_t MAGIC_VAL = 0xbeca;
 static const int DEFAULT_NET = MCTP_NET_ANY;
 static const int DEFAULT_SECONDS_INTERVAL = 2;
 
+static const uint16_t COMMAND_MAGIC = 0x22dd;
+static const uint8_t COMMAND_VERSION = 1;
+
+enum command {
+	COMMAND_RESPONSE = 0x00,
```

Comment:
Will responses to all (future) commands use the same opcode?

(I assume the IID will let us catch mismatched responses, but would a request-specific-response type help to diagnnose?)

Reply:
I was intending to use COMMAND_RESPONSE for all, given IID should be reliable.